app/helpers/items_helper.rb

```diff
@@ -2,7 +2,7 @@
 module ItemsHelper
   def dollar_value(value, addition = '')
     if value.zero?
-      ''
+      '0'
```

Comment:
@edwinthinks Check the history for this -- this MIGHT have been intentionally left blank; I vaguely remember a discussion about this (@seanmarcia ). I want to say it was something along the lines of that the diaper banks were seeing a huge list of `0` in a single column and felt that made it harder to read the data, so we left it out if it was empty.

I don't have any particular feelings about this, myself. If it looks readable in those circumstances then it's :100:

Reply:
I don't remember :(